notebooks/useful_packages.jl

```diff
@@ -0,0 +1,254 @@
+### A Pluto.jl notebook ###
+# v0.15.1
+
+using Markdown
+using InteractiveUtils
+
+# ╔═╡ 539bb62b-e4e3-4d72-837c-302ac9335e2e
+begin
+	using PlutoUI
+	TableOfContents()
+end
+
+# ╔═╡ 27130015-4f45-4ebd-9094-07ec00261e5f
+md"""
+# Useful Packages
+
+Unlike R, Julia does not immediately expose a huge number of functions, but instead requires loading packages (whether from the standard library or from the broader package ecosystem) for a lot of relevant functionality for statistical analysis. There are technical reasons for this, but one further motivation is that Julia is at a broader "technical computing" audience (like MATLAB or perhaps Python) and less at a "statistical analysis" audience. 
+
+This has two important implications:
+1. Even relatively simple programs will often load several packages.
+2. Packages are often focused on adding a relatively narrow set of functionality, which means that "basic" functionality (e.g. reading a CSV file and manipuating it as a DataFrame) is often split across multiple packages. In other words, **see the the first point!**
+
+This notebook is not intended to be an exhaustive list of packages, but rather to highlight a few packages that I suspect will be particularly useful.
+Before getting onto the packages, I have one final hint: take advantage of how easy and first-class package management in Julia is. Having good package management makes reproducible analyses much easier and avoids breaking old analyses when you start a new one. Pluto helpfully installs and manages for you, but the package-manager REPL mode (activated by typing `]` at the `julia>` prompt) is very useful.
+"""
+
+# ╔═╡ cd73a70f-4ce0-4ef8-87be-d3b319066602
+md"""
+
+## Data wrangling
+
+### Reading data
+
+- [Arrow.jl](https://arrow.juliadata.org/dev/manual/) a high performance format for data storage, accessible in R via the `feather` package and in Python via `pyarrow`. This is the format that we store the example and test datasets in for MixedModels.jl.
```

Comment:
I think we should refer to the `arrow` package in R for which the function to use to read an arrow file is, confusingly, called `read_feather`.  There is also a `feather` package but AFAIK it only reads Feather v1 files not Feather v2, a.k.a. Arrow files.

Reply:
Ah, yeah I had forgotten how confusing those names are. :frowning: Thanks for catching that.

Reply:
How about? 

```suggestion
- [Arrow.jl](https://arrow.juliadata.org/dev/manual/) a high performance format for data storage, accessible in R via the [`arrow` package](https://arrow.apache.org/docs/r/) and in Python via `pyarrow`. (Confusingly, the function for reading and writing Arrow format files in R is called `read_feather` and `write_feather`, but the modern Arrow format is distinct from the older Feather format provided by the `feather` package.) This is the format that we store the example and test datasets in for MixedModels.jl.
```